static/app/gettingStartedDocs/node/awslambda.tsx

```diff
@@ -10,71 +14,177 @@ import {
   getCrashReportModalConfigDescription,
   getCrashReportModalIntroduction,
 } from 'sentry/components/onboarding/gettingStartedDoc/utils/feedbackOnboarding';
+import {AwsLambdaArn} from 'sentry/gettingStartedDocs/node/awslambdaArnSelector';
 import {t, tct} from 'sentry/locale';
-import {trackAnalytics} from 'sentry/utils/analytics';
 import {
   getInstallConfig,
   getNodeProfilingOnboarding,
   getSdkInitSnippet,
 } from 'sentry/utils/gettingStartedDocs/node';
-import {
-  InstallationMode,
-  platformOptions,
-} from 'sentry/views/onboarding/integrationSetup';
+
+export enum ModuleFormat {
+  CJS = 'cjs',
+  ESM = 'esm',
+}
+
+export const platformOptions = {
+  moduleFormat: {
+    label: t('Module Format'),
+    items: [
+      {
+        label: t('Lambda Layer (CJS)'),
+        value: ModuleFormat.CJS,
+      },
+      {
+        label: t('NPM Package (ESM)'),
+        value: ModuleFormat.ESM,
+      },
+    ],
+    defaultValue: ModuleFormat.CJS,
+  },
+} satisfies BasePlatformOptions;
 
 type PlatformOptions = typeof platformOptions;
 type Params = DocsParams<PlatformOptions>;
 
+const getEnvSetupSnippet = (params: Params) => `
+NODE_OPTIONS="-r @sentry/aws-serverless/awslambda-auto"
+SENTRY_DSN="${params.dsn.public}"
+${params.isPerformanceSelected ? 'SENTRY_TRACES_SAMPLE_RATE=1.0' : ''}
+`;
+
+const getVerifySnippet = () => `
+exports.handler = Sentry.wrapHandler(async (event, context) => {
+  throw new Error("This should show up in Sentry!")
+});`;
```

Comment:
This needs a CJS and ESM version I'm afraid.

CJS: no need to add the `wrapHandler` function - this is applied by the layer automatically. 
ESM: `exports.handler` should be replaced with `export const handler = ...`